llm_on_ray/ui/start_ui.py

```diff
@@ -1709,14 +1716,13 @@ def _init_ui(self):
     args = parser.parse_args()
 
     file_path = os.path.abspath(__file__)
-    infer_path = os.path.dirname(file_path)
-    repo_path = os.path.abspath(infer_path + os.path.sep + "../")
+    ui_path = os.path.dirname(file_path)
+    repo_path = os.path.abspath(ui_path + os.path.sep + "../")
     default_data_path = os.path.abspath(
-        infer_path + os.path.sep + "../examples/data/sample_finetune_data.jsonl"
+        repo_path + os.path.sep + "../examples/data/sample_finetune_data.jsonl"
     )
 
-    sys.path.append(repo_path)
-    from finetune.finetune import get_accelerate_environment_variable
+    from llm_on_ray.finetune.finetune import get_accelerate_environment_variable
```

Comment:
could you import finetune module to __init__, and import like this:
```suggestion
    from llm_on_ray.finetune import get_accelerate_environment_variable
```